Solutions/MarkLogicAudit/DataConnectors/Connector_MarkLogicAudit.json

```diff
@@ -0,0 +1,167 @@
+{
+    "id": "MarkLogic",
+    "title": "MarkLogicAudit",
+    "publisher": "MarkLogic",
+    "descriptionMarkdown": "MarkLogic data connector provides the capability to ingest [MarkLogicAudit](https://www.marklogic.com/) logs into Microsoft Sentinel. Refer to [MarkLogic documentation](https://docs.marklogic.com/guide/getting-started) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **MarkLogicAudit** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-marklogicaudit-parser)",
```

Comment:
Please validate and proper link as it is redirecting to bing https://aka.ms/sentinel-marklogicaudit-parser

Reply:
@mkchiliveri, this is a link for the master branch. This link isn't supposed to be working yet. I was told that it will work after changes in the master branch.